odoo/addons/base/tests/test_base_document_layout.py

```diff
@@ -206,7 +206,7 @@ def test_parse_company_colors_grayscale(self):
             with Image.open(os.path.join(dir_path, 'logo_ci.png'), 'r') as img:
                 base64_img = image_to_base64(img, 'PNG')
                 doc_layout.logo = base64_img
-            self.assertEqual('#8d8d8d', doc_layout.primary_color)
+            self.assertNotEqual(None, doc_layout.primary_color)
```

Comment:
For style and readability, the two arguments of `assertNotEqual` should be reversed, you want `None` as rightmost argument.

Reply:
Noted, thank you!